pkg/controller.v2/controller_tfjob.go

```diff
@@ -11,10 +11,11 @@ import (
 )
 
 const (
-	failedMarshalTFJobReason = "FailedMarshalTFJob"
-	terminatedTFJobReason    = "TFJobTerminated"
+	failedMarshalTFJobReason  = "FailedMarshalTFJob"
+	terminatedTFJobReason     = "TFJobTerminated"
+	FailedDeletePdbReason     = "FailedDeletePdb"
```

Comment:
We do not to keep the variables public, IMO

Reply:
Ok, I will revised it!
And I found I have some bug in my code, I will also fix them first

Reply:
Thanks!

Reply:
@gaocegege  I think I have corrected all the small bugs, thanks